client/lib/widgets/detail/youtube.dart

```diff
@@ -0,0 +1,38 @@
+import 'package:flutter/material.dart';
+import 'package:youtube_player_flutter/youtube_player_flutter.dart';
+
+class YoutubeWidget extends StatefulWidget {
+  final String youtubeId;
+
+  const YoutubeWidget({required this.youtubeId});
```

Comment:
무조건 Key가 필요. 재사용에 관한 것으로 성능과 직결됨니다

```suggestion
  const YoutubeWidget({Key? key, required this.youtubeId}) : super(key: key);
```

Reply:
이 부분은 참고될만한 자료를 던져주시면 감사하겠습니다